proxy-robots-module/src/main/java/orwell/proxy/config/ConfigFactory.java

```diff
@@ -17,7 +17,7 @@ public ConfigFactory(final ConfigFactoryParameters configFactoryParameters) {
         logback.debug("IN");
         final Configuration configuration = new Configuration(configFactoryParameters);
 
-        if (false == configuration.isPopulated) {
+        if (!configuration.isPopulated) {
```

Comment:
This is a bit redundant to keep the comparison like this (with explicit boolean), and I try to remain consistent in the code (at least for that :D).
Setting the objects to null is required since they are final fields of the class and therefore must be explicitly initialized in the constructor.